packages/zowe-explorer/__tests__/__unit__/job/ZosJobsProvider.unit.test.ts

```diff
@@ -238,11 +238,10 @@ describe("ZosJobsProvider unit tests - Function getChildren", () => {
         favProfileNode.contextValue = globals.FAV_PROFILE_CONTEXT;
         testTree.mFavorites.push(favProfileNode);
 
-        const loadProfilesForFavoritesSpy = jest.spyOn(testTree, "loadProfilesForFavorites");
-
+        mocked(vscode.window.showErrorMessage).mockResolvedValueOnce({ title: "Yes" });
         await testTree.getChildren(favProfileNode);
-
-        expect(loadProfilesForFavoritesSpy).toHaveBeenCalledWith(log, favProfileNode);
+        expect(showErrorMessageSpy).toBeCalledTimes(1);
+        showErrorMessageSpy.mockClear();
```

Comment:
This particular unit test is supposed to check that `getChildren` will call `loadProfilesForFavorites` with the correct parameters given a certain context. It is not supposed to test the error message you added in this PR, and the `loadProfilesForFavorites` function doesn't actually need to be executed. (The fact that it is being executed here has more to do with how the test was originally written (probably by me...) as opposed anything you did.) To avoid unnecessarily running `loadProfilesForFavorites` and hitting the error condition, we can just mock the implementation of `loadProfilesForFavorites` instead of changing what the unit test is checking.

Try putting this test back to the way it was before, and then modifying the `loadProfilesForFavoritesSpy` as follows:

```
       const loadProfilesForFavoritesSpy = jest
            .spyOn(testTree, "loadProfilesForFavorites")
            .mockImplementationOnce(() => Promise.resolve([]));
```

This mock will just pretend that `loadProfilesForFavorites` ran and returned an empty array, so none of the actual code inside `loadProfilesForFavorites` will be executed.